nss-day-cohort-60/national-parks-client
# Md navbar setup

# Description
Created a basic nav bar with login/logout links
Added NavBar.css for styling

# Steps to test

1. Fetch the md-navbar-aetup branch and switch to it in national-parks-client
2. Start/restart debugger in national-parks-server
3. nav bar should display at the top of the current login page with the links: Home, Blog and Login
4. log into site with any user email [ex. frodo@baggins.com]
5. nav bar should appear after user logs in and the logout link should be displayed instead of login link
